include/ddprof_context.hpp

```diff
@@ -36,6 +36,7 @@ struct DDProfContext {
     bool remote_symbolization{false};
     bool disable_symbolization{false};
     bool reorder_events{false}; // reorder events by timestamp
+    int maximum_pids{0};
```

Comment:
It could be convenient to have a default value that reflects "no pid limit"

Reply:
Done